### ISSUE TITLE
[Feature Request] - Disable HumanWalkSniping when CatchLimit is reached

### ISSUE THREAD

**Issue by emwedk** [NoxxDev/NecroBot#778]
Is it possible to make a feature that will disable the HumanWalkSnipe when the CatchLimit has reached.

Purely to increase XP gathered from Pokestops aswell as stocking up on Pokeballs.
With the HumanWalkSnipe enabled it will check fewer Pokestops per hour.

**Comment by masterpowers** [comment]
We are already discussing it here join us https://github.com/NoxxDev/NecroBot/issues/742

**Comment by emwedk** [comment]
not really the same since #742 's main purpose is to stock up on pokeballs
whilst this is to maximize XP outcome and getting the most out of the daily PokestopLimit

**Comment by CreativiTimothy** [comment]
Good idea. Automatically disable sniping once catch limit is reached. This will help with Pokestop farming a lot

DisableSnipingAfterCatchLimit: true/false
